From 73e526d464d0c0970083605269effbff5448b42a Mon Sep 17 00:00:00 2001
From: Huy Bui <huy.bui.wm@renesas.com>
Date: Tue, 27 May 2025 05:49:42 +0700
Subject: [PATCH 13/46] Revert "gpio: gpio-rcar: Temporarily update offset
 resgister for X5H"

This reverts commit 797eb0dd3a933431889edaeb4d05073ba3bf38b7.
---
 drivers/gpio/gpio-rcar.c | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/drivers/gpio/gpio-rcar.c b/drivers/gpio/gpio-rcar.c
index d6f5ad1ce5c2..085cf8b46781 100644
--- a/drivers/gpio/gpio-rcar.c
+++ b/drivers/gpio/gpio-rcar.c
@@ -53,17 +53,17 @@ struct gpio_rcar_priv {
 #define IOINTSEL	0x00	/* General IO/Interrupt Switching Register */
 #define INOUTSEL	0x04	/* General Input/Output Switching Register */
 #define OUTDT		0x08	/* General Output Register */
-#define INDT		0x1c	/* General Input Register */
-#define INTDT		0x80	/* Interrupt Display Register */
-#define INTCLR		0x84	/* Interrupt Clear Register */
-#define INTMSK		0x88	/* Interrupt Mask Register */
-#define MSKCLR		0x8c	/* Interrupt Mask Clear Register */
-#define POSNEG		0x90	/* Positive/Negative Logic Select Register */
-#define EDGLEVEL	0x94	/* Edge/level Select Register */
-#define FILONOFF	0x98	/* Chattering Prevention On/Off Register */
-#define OUTDTSEL	0x0C	/* Output Data Select Register */
-#define BOTHEDGE	0xbc	/* One Edge/Both Edge Select Register */
-#define INEN		0x18	/* General Input Enable Register */
+#define INDT		0x0c	/* General Input Register */
+#define INTDT		0x10	/* Interrupt Display Register */
+#define INTCLR		0x14	/* Interrupt Clear Register */
+#define INTMSK		0x18	/* Interrupt Mask Register */
+#define MSKCLR		0x1c	/* Interrupt Mask Clear Register */
+#define POSNEG		0x20	/* Positive/Negative Logic Select Register */
+#define EDGLEVEL	0x24	/* Edge/level Select Register */
+#define FILONOFF	0x28	/* Chattering Prevention On/Off Register */
+#define OUTDTSEL	0x40	/* Output Data Select Register */
+#define BOTHEDGE	0x4c	/* One Edge/Both Edge Select Register */
+#define INEN		0x50	/* General Input Enable Register */
 
 #define RCAR_MAX_GPIO_PER_BANK		32
 
-- 
2.34.1

